src/view/components/Drawer/index.tsx

```diff
@@ -10,17 +10,19 @@ import DrawerContents from './DrawerContents';
 import { Route } from '../../navigation/navigator';
 
 /** Drawer OwnProps */
-export interface OwnProps {}
+export interface OwnPropsType {}
 
 /** Drawer State */
 export interface OwnState {}
 
-interface StoreProps {
+interface StorePropsType {
   open: boolean;
   locked: boolean;
+  // name: string;
```

Comment:
is this for later? do we need to add TODOs?

Reply:
Yeah I should probably add a TODO thx